src/extension.js

```diff
@@ -31,10 +32,18 @@ Extension.getIcon = Utils.getIcon;
  */
 const ServiceIndicator = GObject.registerClass({
     GTypeName: 'GSConnectServiceIndicator',
-}, class ServiceIndicator extends PanelMenu.SystemIndicator {
+}, class ServiceIndicator extends QuickSettings.QuickMenuToggle {
```

Comment:
```suggestion
}, class ServiceToggle extends QuickSettings.QuickMenuToggle {
```

Renaming for consistency (ServiceIndicator, ServiceToggle).